apps/web/src/components/team-builder/v2/builder.module.css

```
@@ -934,27 +949,127 @@
-    auto              /* type pill */
-    auto              /* cat icon */
-    minmax(0, 11rem)  /* name */
-    auto              /* BP */
-    auto              /* Acc */
-    minmax(0, 1fr)    /* description */
-    auto;             /* calc pill / hint (empty when not applicable) */
+    max-content        /* type + category */
+    minmax(0, 1fr)     /* name — fills space, shrinks to 0 (no overflow) */
+    56px               /* BP */
+    72px;              /* Acc */
   align-items: center;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)